Description: Fix xerox_mfp color mode
 Corrected color mode malfunction.
 
 Bug was introduced by mass change to SANE_VALUE macro, (commit ff048f41760c74e67dfea36a079c71fdb72bbf4c).
Author: Alex Belkin <abc@telekom.ru>
Origin: upstream, http://git.debian.org/?p=sane/sane-backends.git;a=commitdiff;h=043cc4b1651fe602f1b7fb58e6592bd3bef0f4ac
Bug: https://alioth.debian.org/tracker/?func=detail&atid=410366&aid=312485&group_id=30186
Bug-Debian: http://bugs.debian.org/583789

Index: sane-backends-1.0.21/backend/xerox_mfp.c
===================================================================
--- sane-backends-1.0.21.orig/backend/xerox_mfp.c	2010-04-05 15:18:05.000000000 +0200
+++ sane-backends-1.0.21/backend/xerox_mfp.c	2010-06-01 19:34:07.350975337 +0200
@@ -386,6 +386,7 @@
 
 static SANE_String_Const scan_modes[] = {
   SANE_VALUE_SCAN_MODE_LINEART,
+  SANE_VALUE_SCAN_MODE_HALFTONE,
   SANE_VALUE_SCAN_MODE_GRAY,
   SANE_VALUE_SCAN_MODE_COLOR,
   NULL
